tests/smoke-tests/--addon-location.test.ts

```diff
@@ -20,7 +20,7 @@ describe('--addon-location', () => {
     tmpDir = await createTmp();
 
     let { name } = await createAddon({
-      args: [`--addon-location=${addonLocation}`, '--pnpm=true'],
```

Comment:
ember-cli 5.4 doesn't support X=true, and it turns out it wasn't needed, even for 5.3